src/scala/org/pantsbuild/scoverage/report/Settings.scala

```diff
@@ -0,0 +1,67 @@
+package org.pantsbuild.scoverage.report
+
+/**
+ * All parsed command-line options.
+ */
+case class Settings(
+  loadDataDir: Boolean = false,
+  measurementsDirPath: String = "",
+  reportDirPath: String = "",
+  sourceDirPath: String = ".",
+  dataDirPath: String = "",
+  writeHtmlReport: Boolean = true,
+  writeXmlReport: Boolean = true,
+  writeXmlDebug: Boolean = false,
+  cleanOldReports: Boolean = true,
+  targetFilters: Seq[String] = Seq())
+
+object Settings {
+
+  val parser1 = new scopt.OptionParser[Settings]("scoverage") {
+    head("scoverageReportGenerator")
+
+    help("help")
+      .text("Print this usage message.")
+
+    opt[Unit]("loadDataDir")
+      .action((_, s: Settings) => s.copy(loadDataDir = true))
+      .text("Load a single measurements directory instead of aggregating coverage reports. Must pass in `dataDirPath <dir>`")
+
+    opt[String]("measurementsDirPath")
+      .action((dir: String, s: Settings) => s.copy(measurementsDirPath = dir))
+      .text("Directory where all scoverage measurements data is stored.")
+
+    opt[String]("reportDirPath")
+      .action((dir: String, s: Settings) => s.copy(reportDirPath = dir))
+      .text("Target output directory to place the reports.")
+
+    opt[String]("sourceDirPath")
+      .action((dir: String, s: Settings) => s.copy(sourceDirPath = dir))
+      .text("Directory containing the project sources.")
```

Comment:
"Project sources" is pretty ambiguous... is this tool used per-target?

Reply:
No, its just there for backwards compatibility. I don't think there is any use for this tool for latest ReportWriters (or in my report generator): 
https://github.com/scoverage/scalac-scoverage-plugin/blob/9bb6198cb6c4dc6498d034ba4c1810ea2ac8d62c/scalac-scoverage-plugin/src/main/scala/scoverage/report/ScoverageHtmlWriter.scala#L13-L22